static/templates/compose.hbs

```diff
@@ -82,6 +82,9 @@
                                 <span id="compose-globe-icon">
                                     <i class="zulip-icon zulip-icon-globe" title="{{t 'This is a web-public stream' }}" aria-hidden="true"></i>
                                 </span>
+                                <a role="button" class="narrow_to_compose_recipients zulip-icon zulip-icon-arrow-left-circle order-1" data-tooltip-template-id="narrow_to_compose_recipients_tooltip" tabindex="0">
+                                    <template id="narrow_to_compose_recipients_tooltip">{{t 'Narrow to compose recipient' }} <span class="hotkey-hint">({{narrow_to_compose_recipients_key_html}})</span></template>
```

Comment:
`Narrow to message recipient` would be better.

Reply:
Ok, changed the tooltip wording to that.